components/features/galleryAssets/web/buy-action.tsx

```
@@ -0,0 +1,170 @@
+	) => Promise<BuyAssetCompleteResponse | null>;
+	buyStartData: BuyAssetStartResponse | null | undefined;
+	buyCompleteData: BuyAssetCompleteResponse | null | undefined;
+}) => {
+	const { profile } = useAuth();
+
+	const handleBuyStart = async () => {
+		try {
+		} catch (e) {
+			console.error(e);
+			return null;
+		}
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)